etc/nginx.conf

```diff
@@ -0,0 +1,38 @@
+worker_processes auto;
+daemon off;
+pid /tmp/nginx.pid;
+error_log  /dev/stderr;
```

Comment:
Did you notice that master is different: https://github.com/aws/sagemaker-containers/blob/master/src/container_support/etc/nginx.conf#L4

Reply:
this one is on purpose, it avoids doing the symlink shenanigans to get the error log to stderr.

Reply:
👍